app/javascript/controllers/pagination_controller.js

```diff
@@ -0,0 +1,7 @@
+import ApplicationController from './application_controller'
+
+export default class extends ApplicationController {
+  beforeReflex() {
+    window.scrollTo(0, 0)
+  }
+}
```

Comment:
Should this be in an `afterReflex` instead? Or does it work better in `before`?

Reply:
It works better in before for whatever reason.